npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -19,7 +20,11 @@ TODO(jungukcho)
 (https://kubernetes.io/docs/concepts/services-networking/network-policies/#targeting-a-namespace-by-its-name)
 */
 
-var errUnknownPortType = errors.New("unknown port Type")
+var (
+	errUnknownPortType = errors.New("unknown port Type")
+	// ErrUnsupportedTranslationFeature is returned when translation feature is not supported.
+	ErrUnsupportedTranslationFeature = errors.New("unsupported Windows translation feature")
```

Comment:
nitpick: slightly better to have specific error message which features do not support. 
For example, 
```go
ErrUnsupportedNamedPort = errors.New("unsupported namedport on windows ")
ErrUnsupportedNegativeMatch= errors.New("unsupported NotExist on windows ")

```
no strong opinion.